pymatviz/ptable/ptable_matplotlib.py

```diff
@@ -698,6 +812,10 @@ def ptable_scatters(
     symbol_text: str | Callable[[Element], str] = lambda elem: elem.symbol,
     symbol_pos: tuple[float, float] = (0.5, 0.8),
     symbol_kwargs: dict[str, Any] | None = None,
+    # Annotation
+    anno_pos: tuple[float, float] = (0.75, 0.75),
+    anno_text: dict[ElemStr, str] | None = None,
+    anno_kwargs: dict[str, Any] | None = None,
```

Comment:
as above, maybe merge `anno_pos` and `anno_text` into `anno_kwargs`

Reply:
I'm a little bit unsure about this. For the methods of `PTableProjector`, most args are just passed through from the most outer wrap (`ptable_heatmap` -> `projector.add_annotation()` for example), but for these args, perhaps exposing the most frequently used ones might give users a better idea what could be controlled?

Reply:
fair point, feel free to mark this as resolved.

Reply:
Sure thanks!